app/templates/administrative-units/administrative-unit/sites/site.hbs

```diff
@@ -0,0 +1,11 @@
+{{page-title @model.site.address.fullAddress}}
+
+{{breadcrumb
+  this.model.site.address.fullAddress
+  route="administrative-units.administrative-unit.sites"
+  models=(this.model.administrativeUnit.id array this.model.site.id)
```

Comment:
Wait, doesn't this throw an error at the moment? :smile: array is a helper so it should be the first thing there.

`(array this.model.administrativeUnit.id this.model.site.id)`